backend/server/api.go

```diff
@@ -25,64 +24,17 @@ func API(r chi.Router, db *database.Connection, contentStore contentstore.Store,
 	r.Group(func(r chi.Router) {
 		r.Use(middleware.AuthenticateAppAndInjectCtx(db))
 		r.Use(middleware.LogRequests(logger))
+
+		bindSharedRoutes(r, db, contentStore)
 		bindAPIRoutes(r, db, contentStore)
 	})
 }
 
 func bindAPIRoutes(r chi.Router, db *database.Connection, contentStore contentstore.Store) {
-	route(r, "GET", "/operations", jsonHandler(func(r *http.Request) (interface{}, error) {
-		return services.ListOperations(r.Context(), db)
-	}))
-
 	route(r, "GET", "/checkconnection", jsonHandler(func(r *http.Request) (interface{}, error) {
```

Comment:
We'll need /checkconnection available from web.go so clients can verify their configuration.